src/test/java/seedu/clinic/testutil/TypicalSupplier.java

```diff
@@ -80,6 +81,9 @@ public static Clinic getTypicalClinic() {
         for (Supplier supplier : getTypicalSuppliers()) {
             ab.addSupplier(supplier);
```

Comment:
ab used to stand for addressbook, now it has no meaning so i think it should be changed